README.md

```diff
@@ -1,19 +1,58 @@
 # mozilla-infobar
 
+This is the home of the Mozilla, Firefox update, and translation bars. These bars, in the first case, provides an easy way to let your users know that there is a newer version of Firefox available, and that they should upgrade.
+
+In the second instance, it enables you to notify your users when the current page they are viewing, is available in their preferred language.
+
+## Getting the InfoBar
+
+There are three ways to get the code.
+
+### Using Git
+
+First step is to clone the project.
+
+``` bash
+git clone git@github.com:mozilla/mozilla-infobar.git
+```
+
+Once the above operation has finished, you need to copy the following files to your project in the appropriate directories.
+
+``` bash
+css/infobar.css
+js/infobar.js
+```
+
+### Using NPM
+
+You can install it as a NPM module
+
+``` bash
+npm install mozilla-infobar
```

Comment:
I agree if this is to be on npm then I think we should use UMD or similar pattern to support different loaders, else it will likely be a source of confusion for whomever tries to use it.